spec/rails_helper.rb

```diff
@@ -122,17 +122,15 @@
   end
 end
 
-if ENV['travis']
+if ENV['TRAVIS']
```

Comment:
"TRAVIS" and a more generic "CI" variable are defined by default. 
See https://docs.travis-ci.com/user/environment-variables/#default-environment-variables